### PR TITLE
add: custom 404 page

### PR DESCRIPTION
**Pull request by norrec99**
Before
image

After
image

### CODE REVIEW

File: web-app/pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+export default function Example() {
```
Comment (ryancwalsh) [inline web-app/pages/404.tsx:1]: Please change from `Example` to something more relevant.

Comment (norrec99) [reply web-app/pages/404.tsx:1]: done

File: web-app/pages/404.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <div className="bg-white px-4 py-16 sm:px-6 sm:py-24 sm:h-screen md:grid md:place-content-center lg:px-8">
+      <div className="max-w-max mx-auto">
+        <main className="sm:flex">
+          <p className="text-4xl font-extrabold text-purple-600 sm:text-5xl">404</p>
+          <div className="sm:ml-6">
+            <div className="sm:border-l sm:border-gray-200 sm:pl-6">
+              <h1 className="text-4xl font-extrabold text-gray-900 tracking-tight sm:text-5xl">Page not found</h1>
+              <p className="mt-1 text-base text-gray-500">Please check the URL in the address bar and try again.</p>
```
[the remark this answers is not in the document]
Comment (norrec99) [reply web-app/pages/404.tsx:10]: done

File: web-app/pages/404.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-export default function Example() {
+export default function handle404() {
```
Comment (ryancwalsh) [inline web-app/pages/404.tsx:1]: Even though it looks like we're defining a function here, and functions usually have verbs in their names (like you did with `handle404`), this is a special type of function: a React component.

So it should just be a noun.

And I think generally we'll want to use PascalCase for component names. 

https://airbnb.io/javascript/react/#naming

So maybe `ErrorPage404` would be good to try.

**Comment by ryancwalsh** [comment]
@norrec99 Unfortunately I'm not seeing this work yet on my local machine or even on production after merging to main: 

image


https://near-certification-tools-tpq1.onrender.com/xxx

image

Can you please look into it and figure out why they don't look like yours?

**Comment by norrec99** [comment]
You said "after merging to main" but I can't see any changes on the "main branch"

I can see the changes on the "develop branch"
image

but when I switch to the "main branch", the last change was made 13 days ago.
image

It seems to me something went not right while merging "develop" into "main"


Can you see the the new 404 page when you are on the "develop branch"?

**Comment by ryancwalsh** [comment]
@norrec99 This was my mistake! My apologies!

**Pull request merged**